resqpy/grid/_grid.py

```diff
@@ -506,7 +492,10 @@ def off_handed(self):
         """Returns False if IJK and xyz have same handedness, True if they differ."""
 
         ijk_right_handed = self.extract_grid_is_right_handed()
-        assert rqet.find_tag_text(self.crs_root, 'ProjectedAxisOrder').lower() == 'easting northing'
+        if self.crs is None:
+            assert self.crs_uuid is not None
```

Comment:
A general comment, but I much prefer to see a message after assertions like this to aid debugging if the situation occurs.